.gitignore

```diff
@@ -277,3 +277,4 @@ BenchmarkDotNet.Artifacts
 web/
 web-src.*
 MediaBrowser.WebDashboard/jellyfin-web/
+/Common/Networking
```

Comment:
What's this entry?

Reply:
Can ignore it - been having local GIT issues via visual studio.

This change should have been 1 file - commits were stuck locally with no way to sync.

Reply:
My local GIT is screwed. Please ignore it.